src/inspector/injected-script.cc

```diff
@@ -1137,7 +1137,8 @@ Response InjectedScript::bindRemoteObjectIfNeeded(
         inspectedContext ? inspectedContext->getInjectedScript(sessionId)
                          : nullptr;
     if (!injectedScript) {
-      return Response::ServerError("Cannot find context with specified id");
+      // return Response::ServerError("Cannot find context with specified id");
+      return Response::ServerError("[RUN-2486-2498] Cannot find context with specified id A");
```

Comment:
Just calling recordreplay::Print instead seems simpler than messing with these error messages.  That will also work better as more diagnostics get added.

Reply:
`Print` works but is basically impossible to track or triage and thus undesirable. We are currently working on better mechanisms to track failures and problems, as part of the `Failure Tracking` project. Returning the message allows the caller to report it in a way that is trackable and/or triagable.

If we don't want to return, we can instead first land [RUN-2499](https://linear.app/replay/issue/RUN-2499/add-command-failure-diagnostic-mechanism-that-is-triageable) to provide a new mechanism to attach diagnostics to a command handler result (even if its not fully hooked up in its first version) 🤷

Reply:
If you think `RUN-2499` is a good idea, I will move forward with it first and change this PR to use those new diagnostic functions.

Reply:
Eh, you can look in the controller logs when this error message shows up and see everything that has been printed.  Not everything needs to be included in crash reports.  Doing RUN-2499 would be fine -- it seems like more work than necessary at this point, but will be helpful to have in the long term.

Reply:
I guess, we think that RUN-2499 is a good idea because:

1. If we don't have a proper system in place to track "diagnostic hits" (similar to how we currently track Assert hits), we rely on STRs - which are not always possible, as we have seen in quite a few command failure cases already.
2. Also, command failures are going to be separate from command crashes. So crash reports would not necessarily be generated for these issues.